app/inpututils.cpp

```diff
@@ -270,6 +270,40 @@ bool InputUtils::cpDir( const QString &srcPath, const QString &dstPath )
   return true;
 }
 
+QString InputUtils::renameFile( const QString &srcPath, const QString &prefix )
+{
+  QFile file( srcPath );
+
+  if ( file.exists() )
+  {
+    QFileInfo info( srcPath );
+    QString newFilename;
+    QString currTime = QDateTime::currentDateTime().toUTC().toString( "yyyy-MM-dd_hh_mm_ss.zzz" );
```

Comment:
I think we can skip milliseconds, no need for so much precision :) Also I would simplify the filenames to something like YYMMDD-HHMMSS

Finally, I would use local time instead of UTC... I think that's more natural...